src/wmtk/SimplicialComplex.cpp

```diff
@@ -417,6 +417,35 @@ std::vector<Simplex> SimplicialComplex::vertex_one_ring(const Mesh& m, Tuple t)
     return std::vector<Simplex>(vs.begin(), vs.end());
 }
 
+std::vector<Simplex> SimplicialComplex::vertex_one_ring(const TriMesh& m, Tuple t)
```

Comment:
does it make sense to specifiy this is the set of vertices that are one ring of vertices? i always forget these definitions